src/picongpu/include/particles/particleToGrid/ComputeGridValuePerFrame.def

```diff
@@ -77,10 +84,17 @@ public:
                             BoxTmp& tmpBox);
 };
 
-/** Predefined Calculations for \see fieldOutput.param
+/** Density Operation for Particle to Grid Projections
+ *
+ * Derives a scalar density field from a particle species at runtime.
+ * Each value is mapped per cell according to the species' spatial shape.
+ *
+ * @tparam T_Species a @see picongpu::Particles class with a species definition,
+ *                   see @see speciesDefinition.param
+ *
+ * @typedef CreateDensityOperation< T_Species >::type a field that can be used
+ *          in @see fileOutput.param @see picongpu::FileOutputFields
  */
-
-/* Density */
```

Comment:
it's not removed, there is a whole new block now with a real doxygen comment for this class.